src/resolver/resolver.zig

```diff
@@ -3408,7 +3408,10 @@ pub const Resolver = struct {
                             // with this same path. The goal of this code is to avoid having
                             // both the "module" file and the "main" file in the bundle at the
                             // same time.
-                            if (kind != ast.ImportKind.require) {
+                            //
+                            // Additionally, if this is for the runtime, use the "main" field.
+                            // If it doesn't exist, the "module" field will be used.
+                            if (!r.opts.runtime and kind != ast.ImportKind.require) {
```

Comment:
Since this is only used by the Resolver, maybe we could make this a flag inside Resolver like `enable_module_field`?

Reply:
and then default it to true but set it to false in VirtualMachine.init

Reply:
good idea, updated